src/components/MultiHopTrade/components/TradeConfirm/hooks/useThorStreamingProgress.tsx

```diff
@@ -54,10 +55,13 @@ const getStreamingSwapMetadata = (
       }),
     ) ?? []
 
+  const successfulSwapCount = (data.count ?? 0) - (failedSwaps.length ?? 0)
```

Comment:
I know this logic is taken from existing but it doesn't look correct from a user perspective:
<img width="433" alt="image" src="https://github.com/user-attachments/assets/6160732e-2592-4685-a39a-0634f5f33d15" />

- `count` includes failed swaps 
- we don't use `failedSwaps` anywhere

If the goal is to displayed the "current swaps out of total" for the sake of showing a progress, couldn't we simply show `count`?
With the current way, failed swaps won't contribute to the total, but in reality, they do, regardless of whether they're fail or success swaps

Reply:
I removed it in favor of `attemptedSwapCount` (which is `count` in reality)